extensions/adobe/experience/offer-management/fallback-offer.schema.json

```diff
@@ -0,0 +1,80 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id":
+    "https://ns.adobe.com/experience/offer-management/fallback-offer",
+  "meta:extends": [
+    "https://ns.adobe.com/experience/offer-management/offer"
+  ],
+  "type": "object",
+  "title": "Adobe Experience Cloud Fallback Offer",
```

Comment:
Is this really XC or specific to some part of it?

Reply:
updated